src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,111 @@
+import calculate from '../logic/calculate';
+
+describe('Start tests', () => {
+  test('8 - 4 = 4', () => {
+    let item = {};
+
+    item = { ...item, ...calculate(item, '8') };
```

Comment:
- [ ] You have the calculate test in both `tests` and `__tests__` kindly remove one of the test files.